### PR TITLE
Update Appledaily.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Appledaily.xml [hunk 1]
```diff
@@ -40,6 +42,8 @@
 	<target host="staticlayout.appledaily.hk" />
 	<target host="video.appledaily.com.hk" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Appledaily.xml:43]: There are more possible targets:

```
	<target host="appledaily.com" />
	<target host="20.appledaily.com" />
	<target host="64.appledaily.com" />
	<target host="add.appledaily.com" />
	<target host="arc-static.appledaily.com" />
	<target host="as.appledaily.com" />
	<target host="auth.appledaily.com" />
	<target host="breastcancer.appledaily.com" />
	<target host="cmpg.appledaily.com" />
	<target host="cms-privilegehk.appledaily.com" />
	<target host="corporationplan.appledaily.com" />
	<target host="dce2019.appledaily.com" />
	<target host="dev-arc-static.appledaily.com" />
	<target host="dev-cmpg.appledaily.com" />
	<target host="dev-evt.appledaily.com" />
	<target host="dev-policy.appledaily.com" />
	<target host="dev-vtrk.appledaily.com" />
	<target host="hk.entertainment.appledaily.com" />
	<target host="qa-tw.entertainment.appledaily.com" />
	<target host="tw.entertainment.appledaily.com" />
	<target host="uat-tw.entertainment.appledaily.com" />
	<target host="events.appledaily.com" />
	<target host="evt.appledaily.com" />
	<target host="dev-tw.feature.appledaily.com" />
	<target host="hk.feature.appledaily.com" />
	<target host="qa-hk.feature.appledaily.com" />
	<target host="tw.feature.appledaily.com" />
	<target host="uat-hk.feature.appledaily.com" />
	<target host="uat-tw.feature.appledaily.com" />
	<target host="hk.finance.appledaily.com" />
	<target host="qa-tw.finance.appledaily.com" />
	<target host="tw.finance.appledaily.com" />
	<target host="uat-tw.finance.appledaily.com" />
	<target host="fruitgym.appledaily.com" />
	<target host="gift.appledaily.com" />
	<target host="green.appledaily.com" />
	<target host="hk.appledaily.com" />
	<target host="imp.appledaily.com" />
	<target host="tw.inv.appledaily.com" />
	<target host="hk.lifestyle.appledaily.com" />
	<target host="qa-tw.lifestyle.appledaily.com" />
	<target host="tw.lifestyle.appledaily.com" />
	<target host="uat-tw.lifestyle.appledaily.com" />
	<target host="luckydraw-100k.appledaily.com" />
	<target host="myqrcode.appledaily.com" />
	<target host="news.appledaily.com" />
	<target host="hk.news.appledaily.com" />
	<target host="qa-tw.news.appledaily.com" />
	<target host="tw.news.appledaily.com" />
	<target host="uat-tw.news.appledaily.com" />
	<target host="policy.appledaily.com" />
	<target host="privilegehk.appledaily.com" />
	<target host="privilegehk-static.appledaily.com" />
	<target host="privilegetw.appledaily.com" />
	<target host="privilegetw-static.appledaily.com" />
	<target host="qa-policy.appledaily.com" />
	<target host="quote.appledaily.com" />
	<target host="realtime.appledaily.com" />
	<target host="reporter.appledaily.com" />
	<target host="hk.sports.appledaily.com" />
	<target host="qa-tw.sports.appledaily.com" />
	<target host="tw.sports.appledaily.com" />
	<target host="uat-tw.sports.appledaily.com" />
	<target host="student.appledaily.com" />
	<target host="tw.ticket.appledaily.com" />
	<target host="tw.appledaily.com" />
	<target host="tw-survey.appledaily.com" />
	<target host="uat-arc-static.appledaily.com" />
	<target host="uat-auth.appledaily.com" />
	<target host="uat-corporationplan.appledaily.com" />
	<target host="uat-fruitgym.appledaily.com" />
	<target host="uat-policy.appledaily.com" />
	<target host="uat-student.appledaily.com" />
	<target host="hk.video.appledaily.com" />
	<target host="tw.video.appledaily.com" />
	<target host="vtrk.appledaily.com" />
	<target host="worldcup2014.appledaily.com" />
	<target host="xinjiangcamps.appledaily.com" />
```

File: src/chrome/content/rules/Appledaily.com.hk.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Non-functional hosts
+		SSL peer certificate was not OK:
+		- appledaily.com.hk
+-->
+<ruleset name="Appledaily.com.hk">
+	<target host="video.appledaily.com.hk" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Appledaily.com.hk.xml:7]:
```suggestion
	<target host="64.appledaily.com.hk" />
	<target host="mlprd.api.appledaily.com.hk" />
	<target host="as.appledaily.com.hk" />
	<target host="auth.appledaily.com.hk" />
	<target host="charity.appledaily.com.hk" />
	<target host="goodest-api.appledaily.com.hk" />
	<target host="goodest-cms.appledaily.com.hk" />
	<target host="luckydraw-100k.appledaily.com.hk" />
	<target host="mldev-api.appledaily.com.hk" />
	<target host="mlprd-api.appledaily.com.hk" />
	<target host="mluat-api.appledaily.com.hk" />
	<target host="news.appledaily.com.hk" />
	<target host="racing.appledaily.com.hk" />
	<target host="statefarm.appledaily.com.hk" />
	<target host="travelseed-cms.appledaily.com.hk" />
	<target host="uat-racing.appledaily.com.hk" />
	<target host="mlprd.ugcapi.appledaily.com.hk" />
	<target host="vdo.appledaily.com.hk" />
	<target host="video.appledaily.com.hk" />
```

**Comment by cschanaj** [comment]
@pipboy96 would you be interested to merge this PR?

**Comment by pipboy96** [comment]
@cschanaj Sorry, I still can't merge PRs.

**Pull request merged**